modules/policy-criteria.adoc

```diff
@@ -35,518 +35,820 @@ To use logical operators `AND`, `OR`, and `NOT` for creating security policies,
 However, on earlier versions you can still use regular expressions for the fields listed in the *Regular expressions* column.
 ====
 
-[cols="<,<,<,^,^,^,<"]
+[cols="<,<,<,<,^,^,^,<"]
 |===
-| *Attribute* | *Description* | *RHACS version* | *Regular expressions* | *NOT* | *AND, OR* | *Phase*
+| *Attribute* | *Description* | *JSON Attribute* | *Allowed Values* | *Regular expressions* | *NOT* | *AND, OR* | *Phase*
 
-| Namespace
-| The name of the namespace.
-| 3.0.51 and newer
-| ✓
-| ✓
-| ✓
-| Deploy
+8+| *Section: Image registry*
 
 | Image Registry
 | The name of the image registry.
-| All
+| Image Registry
+| String
 | ✓
 | ✓
 | ✓
 | Deploy
 
-| Image Remote
+| Image Name
 | The full name of the image in registry, for example `library/nginx`.
-| All
+| Image Remote
+| String
 | ✓
 | ✓
 | ✓
 | Deploy
 
 | Image Tag
 | Identifier for an image.
-| All
+| Image Tag
+| String
 | ✓
 | ✓
 | ✓
 | Deploy
 
-| Days since image was created
-| The number of days from image creation date.
-| All
+| Image Signature
+| The list of signature integrations you can use to verify an image's signature. Create alerts on images that either do not have a signature or their signature is not verifiable by at least one of the provided signature integrations.
+| Image Signature Verified By
+| A valid ID of an already configured image signature integration
+| ✕
+| ✕
+| ! `OR` only
+| Deploy
+
+8+| *Section: Image contents*
+
+| Image age
+| The minimum number of days from image creation date.
+| Image Age
+| Integer (whole numbers only)
 | ✕
 | ✕
 | ✕
 | Build
 
-| Days since image was last scanned
-| The number of days since the last image scan.
-| All
+| Image scan age
+| The minimum number of days since the image was last scanned.
+| Image Scan Age
+| Integer (whole numbers only)
 | ✕
 | ✕
 | ✕
 | Build
 
+| Image User
+| Matches the USER directive in the Dockerfile. See [ Docker documentation Hyperlink] for details
+https://docs.docker.com/engine/reference/builder/#user.
+| Image User
+| String
+| ✓
+| ✓
+| ✓
+| Build
+
 | Dockerfile Line
 | A specific line in the Dockerfile, including both instructions and arguments.
-| All
+| Dockerfile Line
+| One of: LABEL, RUN, CMD, EXPOSE, ENV, ADD, COPY, ENTRYPOINT, VOLUME, USER, WORKDIR, ONBUILD
 | ! only for values
 | ✕
 | ✓
 | Build
 
-| Image is NOT Scanned
-| No scan data is available for the image.
-| All
+| Image scan status
+| Check if an image was scanned..
+| Unscanned Image
+| Boolean
 | ✕
 | ✕
 | ✕
 | Build
 
 | CVSS
 | Common Vulnerability Scoring System, use it to match images with vulnerabilities whose scores are greater than `>`, less than `<`, or equal to `=` the specified CVSS.
-| All
+| CVSS
+| <, >, <=, >= or nothing (which implies equals to) +
```

Comment:
I think you need an escape (`a`) to stop asciidoc from messing with <=, but that may cause it to render `+` exactly. Sometimes a `\` in front of `<=` works, but not sure if it will in a table row.

```suggestion
a| <, >, <=, >= or nothing (which implies equal to) +
```